docs/inferences/model-runner.md

```diff
@@ -0,0 +1,40 @@
+Running inferences with the standard `landingai.predict.Predictor` will send your image to LandingLens cloud, which is ideal if you don't want to worry about backend scalability, hardware provisioning, availability, etc. But this also adds some networking overhead that might limit how many inferences per second you can run.
+
+If you need to run several inferences per second, and you have your own cloud service or local machine, you might want to run inference using your own resources. For that, we provide **[Model Runner](https://support.landing.ai/docs/docker-deploy)**, a Docker image with your LandingLens trained model embeded that you can run anywhere.
+
+
+???+ note
+
+    You can get more details on how to setup and run Model Runner container locally or in your own cloud service in our [Support Center](https://support.landing.ai/docs/docker-deploy).
+
+    Once you go through the Support Center guide, you will have the model running in a container, accessible in a specific host and port, used in the below examples as `localhost` and `8000` respectively.
```

Comment:
Break into 2 sentences and rephrase: "specific host and port. The example below refers to these as `localhost` and `8000`, respectively.

Reply:
Done! Thanks!